src/requirements.txt

```diff
@@ -1,14 +1,14 @@
-Flask==2.1.3
-urllib3==1.26.5
+Flask==3.0.3
+urllib3==1.26.19
 jsonschema==3.2.0
 # The commons package requires requests>=2.22.0 and PyYAML>=5.3.1
-requests==2.27.1
+requests==2.32.0
 PyYAML==5.4.1
 
 # Flask 2.1.3 installs the latest Werkzeug==3.0.0 (released on 9/30/2023) and causing import issues 
 # Use a pinned version 2.3.7 (the latest release before 3.0.0) to mitigate temporaryly
 # Will upgrade Flask to newer version later on across all APIs. 10/3/2023 - Zhou
```

Comment:
The comments on the old `Werkzeug 2.3.7` is no longer relevant to the new version and should be removed